README.md

```diff
@@ -62,10 +62,10 @@ var myApp = angular.module('myModule', ['angular-google-analytics']);
 ### 3- Set your Google Analytics account and start tracking:
 
 ```JavaScript
-myApp.config(function (AnalyticsProvider) {
+myApp.config('AnalyticsProvider', function (AnalyticsProvider) {
```

Comment:
This needs to be: ```['AnalyticsProvider', ... }])```

Note the encapsulating square brackets. That is how DI array declarations work.